static/inline/scss/styles.scss

```diff
@@ -587,10 +587,7 @@ a {
 
 .tvp-video-item-active{
     .tvp-video-play{
-        background-color: #e57211;
-    }
```

Comment:
we hide the 'Play icon' in order to show the 'Now Playing' text on the active video